resources/js/components/user/Add/AddForm.vue

```diff
@@ -0,0 +1,154 @@
+                            ></v-text-field>
+                            <span
+                                style="color: #ff5252; font-size: 12px; padding-left: 33px;"
+                                >{{ errorEmail }}</span
+                            >
+                        </ValidationProvider>
+
+                        <ValidationProvider
+                            v-slot="{ errors }"
+                            name="Password"
+                            rules="required"
+                        >
+                            <v-text-field
+                                id="password"
+                                label="Add Password *"
+                                name="password"
+                                prepend-icon="mdi-lock"
+                                type="password"
+                                v-model="user.password"
+                                :error-messages="errors"
+                                v-on:keyup.enter="onClickAddButton"
+                                outlined
+                                required
+                                autocomplete="off"
+                            />
+                        </ValidationProvider>
+                    </v-form>
+                </v-card-text>
+                <v-card-actions>
+                    <v-spacer />
+                    <v-btn
+                        color="primary"
+                        :disabled="isAddBtnDisabled"
+                        @click="onClickAddButton"
+                        :loading="isBtnLoading"
+                    >
+                        Add User
+                    </v-btn>
+                </v-card-actions>
+            </v-card>
+        </v-col>
+    </v-row>
+</template>
+
+<script>
+import {
+    ValidationProvider,
+    ValidationObserver
+} from "vee-validate/dist/vee-validate.full";
+export default {
+    name: "AddForm",
+    components: {
+        ValidationProvider,
+        ValidationObserver
+    },
+    data() {
+        return {
+            user: {
+                name: "",
+                email: "",
+                password: ""
+            },
+            isBtnLoading: false,
+            errorEmail: ""
+        };
+    },
+    created() {
+        axios.get("/api/users").then(res => {
+            this.users = res.data;
+        });
+    },
+    computed: {
+        isAddBtnDisabled() {
+            if (!this.user.name || !this.user.password || !this.user.email) {
+                return true;
+            } else {
+                return false;
+            }
+        }
+    },
+    methods: {
+        async onClickAddButton() {
+            if (this.isAddBtnDisabled === true) {
+                return false;
+            }
+
+            this.isBtnLoading = true;
+
+            for (let i = 0; i < this.users.length; i++) {
+                if (this.user.email === this.users[i].email) {
+                    window.alert(
```

Comment:
If the `users` table has many records ( ex: 100000 records) you can not get all users to check the existed email.
Please use the another way.